strings/LongestPalindromicSubstring.java

```diff
@@ -0,0 +1,40 @@
+package strings;
+
```

Comment:
Please add proper problem statement so that it become easy for begginers to understand. As the soul purpose of these implementations is for educational purpose.